src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,378 @@
+            "title": "Targeted Locus",
+            "description": "The specific chromosomal region(s) this construct library was designed to target.",
+            "comment": "Each locus is defined using 1-based, closed coordinates for a specific version of the reference genome assembly.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Targeted Locus",
+                "type": "object",
+                "required": [
+                    "assembly",
+                    "chromosome",
+                    "start",
+                    "end"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "assembly": {
+                        "title": "Mapping assembly",
+                        "description": "The genome assembly to which coordinates relate (e.g., GRCh38).",
+                        "type": "string",
+                        "enum": [
+                            "GRCh38",
+                            "GRCm39"
+                        ]
+                    },
+                    "chromosome": {
+                        "title": "Chromosome",
+                        "description": "The number (or letter) designation for the chromosome, e.g. chr1 or chrX",
+                        "type": "string",
+                        "pattern": "^(chr[0-9A-Za-z]+)$"
+                    },
+                    "start": {
+                        "title": "Start",
+                        "description": "The 1-based starting coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "end": {
+                        "title": "End",
+                        "description": "The 1-based ending coordinate.",
+                        "type": "integer",
+                        "minimum": 0
+                    }
+                }
+            }
+        },
+        "plasmid_map": {
+            "title": "Plasmid Map",
+            "description": "An image of the plasmid map used in this construct library.",
+            "type": "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "comment": "See document.json for a list of available identifiers.",
+                "type": "string",
+                "linkTo": "Document"
+            }
+        },
+        "average_size": {
+            "title": "Average Size",
+            "description": "The average size of the inserts (such as guides) cloned into vectors in the library.",
+            "comment": "May not be submitted with size_range.",
+            "type": "number",
+            "minimum": 0
+        },
+        "size_range": {
+            "title": "Size Range",
+            "description": "The numerical range of the size of the inserts (such as guides) cloned in vectors in the library.",
```

Comment:
in general, I think examples (such as) belong in comments. And I am a bit confused about the meaning of size_range for MPRA vs. CRISPR guides.

Reply:
Understood, removed. The language says the size property should describe the insert, which I think should apply to either MPRA or CRISPR guides (and the property is allowed for either)